server/src/RestResourceUser.cc

```diff
@@ -140,17 +140,22 @@ void RestResourceUser::handlerUser(void)
 static void addUserRolesMap(
   FaceRest::ResourceHandler *job, JSONBuilder &agent)
 {
+	using namespace StringUtils;
```

Comment:
Why add this line? In this file, StringUtils class methods are all qualified `StringUtils::`
It seems needless.